#! /usr/bin/env python

PACKAGE='trajectory_generation'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name                	Type      Level     Description                   		Default
gen.add("X_waypoint",       	str_t,    	0,    "set the waypoints X-component",  	"0,20")
gen.add("Y_waypoint",       	str_t,    	0,    "set the waypoints Y-component",  	"20,20")
gen.add("Reset_Xwp",        	bool_t,    	0,    "whether to reset X_waypoints",  		True)
gen.add("Reset_Ywp",        	bool_t,    	0,    "whether to reset Y_waypoints",  		True)
gen.add("Concatenate_Xwp",      bool_t,    	0,    "whether to concatenate X_waypoints", False)
gen.add("Concatenate_Ywp",      bool_t,    	0,    "whether to concatenate Y_waypoints", False)

exit(gen.generate(PACKAGE, "trajectory_generation", "waypointSet"))
